src/renderer/SortWrapper.tsx

```diff
@@ -8,27 +8,25 @@ import {
   useLayoutEffect,
 } from 'react';
 
-interface IBoundingBoxMap {
-  [key: string]: DOMRect;
```

Comment:
Feels bad that you worked on this. At least we never have to look at all this again :D